driver/src/main/java/org/neo4j/driver/internal/async/ExplicitTransaction.java

```diff
@@ -152,7 +126,7 @@ else if ( state == State.ROLLED_BACK )
         }
         else if ( state == State.ROLLED_BACK )
         {
-            return completedWithNull();
+            return failedFuture( new ClientException( "Can't rollback, transaction has been rolled back" ) );
```

Comment:
Same here, and also the following (unchanged) messages with regards to standardisation,